python/tskit/metadata.py

```diff
@@ -550,10 +550,10 @@ def decode_row(self, row: bytes) -> Any:
 
     def encode_row(self, row: bytes) -> Any:
         """
-        Encode an encoded row (dict) of metadata to its binary representation (bytes)
+        Encode a row (dict) of metadata to its binary representation (bytes)
```

Comment:
This contradicts the type annotation, doesn't it?

Reply:
Yes! Need static checking! https://github.com/tskit-dev/tskit/pull/741/commits/bddf72da88a60813cd898830207814a63e059c0c